src/popup.ts

```diff
@@ -147,6 +147,41 @@ async function init() {
     false
   );
 
+  document.body.addEventListener("keydown", (e) => {
+    if (
+      document.activeElement?.nodeName === "INPUT" ||
+      document.activeElement?.nodeName === "TEXTAREA"
```

Comment:
I'm not sure if there's a better way to handle this. But obviously, editable `div` is not included, and read-only input, read-only textarea, radio and checkbox are not excluded.